tls/_common/test/test_prf.py

```diff
@@ -157,3 +163,107 @@ def test_sha384_148bytes(self):
         )
         actual_output = prf(secret, label, seed, hashes.SHA384(), 148)
         assert actual_output == expected_output
+
+
+def ascii_bytes(min_size, max_size):
```

Comment:
Is this PRF-specific? I feel like this might need to go someplace more "common" (or test utils etc.)  if we hope to reuse it. But I think it's safe to leave it as is for now, we'll cross that bridge when we get to it.